database/security-rules.json

```diff
@@ -160,7 +182,7 @@
         "$userActions": {
           ".write": "newData.hasChildren(['publicId','action','timestamp'])",
           "publicId": {
-            ".validate": "newData.isString() && root.child('auth/publicIds').child(newData.val()).val() === auth.uid"
+            ".validate": " true || newData.isString() && root.child('auth/publicIds').child(newData.val()).val() === auth.uid"
```

Comment:
I don't think we should let any user log a userAction for any other user. userActions are intended to be the logs for individual user's actions. If you want to have the logging user include the publicIds of their team members, that might be better than relaxing the security rules.